great_expectations/util.py

```diff
@@ -1590,10 +1591,69 @@ def convert_ndarray_decimal_to_float_dtype(data: np.ndarray) -> np.ndarray:
     return convert_decimal_to_float_vectorized(data)
 
 
-def get_context():
-    from great_expectations.data_context.data_context import DataContext
+def get_context(
+    context_root_dir: Optional[str] = None,
+    runtime_environment: Optional[dict] = None,
+    project_config: Optional[Any] = None,
+    ge_cloud_base_url: Optional[str] = None,
+    ge_cloud_access_token: Optional[str] = None,
+    ge_cloud_organization_id: Optional[str] = None,
```

Comment:
This is a good case for defining some function overloads.
https://mypy.readthedocs.io/en/stable/more_types.html?highlight=overloads#function-overloading

I'd love to pair on this 😄